### PR TITLE
improve(arbitrum-messenger): Emit event including return value of Arbitrum sys contract call

### PR DESCRIPTION
**Pull request by nicholaspai**
**Motivation**

The `relayMessage` function of the `Arbitrum_Messenger` contract does not emit a relevant event after executing a sensitive action. The `relayMessage` function calls as a subroutine `sentTxToL2NoAliasing` which itself returns the `uint256` value `seqNum`, but this return value is not logged in the `relayMessage` function.
Consider emitting events after sensitive changes take place, to facilitate tracking and notify off-chain clients following the contract's activity.

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [X]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/core/contracts/insured-bridge/avm/Arbitrum_Messenger.sol [hunk 1]
```diff
@@ -43,13 +55,25 @@ contract Arbitrum_Messenger is Ownable, Arbitrum_CrossDomainEnabled, MessengerIn
         bytes memory message
     ) external payable override onlyOwner {
         // Since we know the L2 target's address in advance, we don't need to alias an L1 address.
-        sendTxToL2NoAliassing(
+        uint256 seqNumber =
+            sendTxToL2NoAliassing(
+                target,
+                userToRefund,
+                l1CallValue,
+                maxSubmissionCost, // TODO: Determine the max submission cost. From the docs: "current base submission fee is queryable via ArbRetryableTx.getSubmissionPrice"
```
Comment (chrismaree) [inline packages/core/contracts/insured-bridge/avm/Arbitrum_Messenger.sol:63]: shall we remove this todo?

File: packages/core/contracts/insured-bridge/avm/Arbitrum_Messenger.sol [hunk 2]
```diff
@@ -13,6 +13,18 @@ import "@openzeppelin/contracts/access/Ownable.sol";
  * owner for L2 contracts that expect to receive cross domain messages.
  */
 contract Arbitrum_Messenger is Ownable, Arbitrum_CrossDomainEnabled, MessengerInterface {
+    event RelayedMessage(
```
Comment (chrismaree) [inline packages/core/contracts/insured-bridge/avm/Arbitrum_Messenger.sol:16]: this is great!

**Review by chrismaree — APPROVED** [review]
is there any similar info we'd like to add to the optimism messenger?

File: packages/core/contracts/insured-bridge/avm/Arbitrum_Messenger.sol (hunk 1 above)
Comment (nicholaspai) [reply packages/core/contracts/insured-bridge/avm/Arbitrum_Messenger.sol:63]: Done

**Comment by nicholaspai** [comment]
> is there any similar info we'd like to add to the optimism messenger?

Eh the Optimism sys contracts don't return any value, so we'd only be emitting the function params. I think we don't make this change until we test it more in the wild.

**Pull request merged**